.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+
+name: Poetry publish
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  buildPush:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+      - name: Install Poetry
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+      # use the hardcoded version number for now. if this is merged to the main repo, you can uncomment these lines to automatically tag the version number with the github run number
+      # - name: Set Version number
+      #   run: |
+      #     poetry version 0.0.${{ github.run_number }}
+      - name: Build and Publish to PyPI
+        run: |
+          poetry config pypi-token.pypi ${{ secrets.PYPI_API_TOKEN }}
+          poetry publish --build
```

Comment:
newline needed

Reply:
I have also uncommented the poetry version, this will set the current version to 0.1.(GitHub run number), so 0.1.1 with your first run